CHANGELOG.md

```diff
@@ -1,12 +1,23 @@
 # Release History
 
 
-## 1.2.0 (TBD)
+## 1.2.0 (2023-03-02)
+
+### New Features
 
-### New Features:
 - Added support for displaying source code as comments in the generated scripts when registering stored procedures. This
 is enabled by default, turn off by specifying `source_code_display=False` at registration.
+- Added a parameter `if_not_exists` when creating udf/udtf/sproc from snowpark python to ignore creating the specified
```

Comment:
Change this: "...when creating udf/udtf/sproc from snowpark python to ignore creating the specified function/procedure if it already exists."

To this:
"...when creating a UDF, UDTF or Stored Procedure from Snowpark Python to ignore creating the specified function or procedure if it already exists."